api/services/wwwConfig.js

```diff
@@ -149,10 +147,15 @@ function _generateEnvs(bag, next) {
   var who = bag.who + '|' + _generateEnvs.name;
   logger.verbose(who, 'Inside');
 
-  var apiUrl = bag.apiSystemIntegration.data &&
-    bag.apiSystemIntegration.data.url;
+  var publicApiUrl = bag.publicApiIntegration.data &&
+    bag.publicApiIntegration.data.url;
+
+  var internalApiUrl = bag.internalApiIntegration.data &&
```

Comment:
Is `bag.internalApiIntegration` always defined?

Reply:
yeah, it will always be defined